### ISSUE TITLE
update application page & split based on location

### ISSUE THREAD

**Issue by jsms90** [foundersandcoders/our-site#427]
Based on [this proposal](https://github.com/foundersandcoders/selection-committee/issues/53), passed by the Selection Committee

@jsms90 write the 1st draft in a google doc, add everyone to co-edit, then submit PR

> The application page will explain the prerequisites and link to _location-specific_ pages for London and Nazareth, which describe our exact process in each location and link to the location-specific application. If applicants want to apply to both campuses, they can fill out both applications.

  + [x] create new text for apply page - need to have working accounts (FCC, Codewars, GitHub)
  + [x] create new text for prereqs
  + [x] specify C4E & joining gitter channels
  + [x] create location-specific pages for London & Gaza
  + Shireen @shiryz create Arabic and Hebrew versions #434 

---

Applications for Founders & Coders X close (date):
 
_Founders & Coders X is a is a not-for-profit endeavor whose main purpose is to provide study and training programming for the general public and to provide workspace for its students and graduates._
 
_The program is full-time, 5 days each week, for at least 8 hours each day, similar to a full-time job. No academic training in computer science or software engineering is required, just a strong commitment to self-learning and giving back. All backgrounds are welcome!_
 
- Email address
- Do you have the right to work in the UK/Israel?
- Do you plan to seek employment after the course?
- Do you agree to abide by our student agreement? _Link to versions for London/Nazareth/Gaza_ (G-specific - https://goo.gl/fpBgBS)

_All students must agree abide by our student agreement as a condition of accepting an offer to study at Founders & Coders. Graduates of each cohort of students are expected collectively to take the lead on facilitating the programme for the next student cohort. We ask you to consider either seeking employment with one of our partners or, alternatively, making a regular voluntary contribution after you graduate as a way of 'paying it forward', so that we can provide a place for someone in a future cohort.

**Comment by jsms90** [comment]
[Copy](https://docs.google.com/document/d/1HEST7ZghnRXyyYfig6MraGPBY_FKI6UWAQ1ZgGCgez8/edit?ts=594458b6#) from @rebeccaradding

**Comment by rebeccaradding** [comment]
**Questions**:
-  Should the location-specific info pages be merged with the new apply pages (probably putting the logistic info below the application info)?
- Should we remove press links from Our Story? I think they will be more visible on the location-specific Apply pages.

**Other tasks:**
- Remove partner logos from the website (we will put them on the new location-specific application pages, as requested by our Gaza partners)
- Update text of Our Story, to reflect that the Nazareth campus has opened (not sure yet if we can add that we have opened a campus in Gaza)

**Comment by rebeccaradding** [comment]
@jsms90 at this time we do not need a location-specific page for Gaza as GSG will use their own channels to advertise.

**Comment by rebeccaradding** [comment]
@jsms90 once we are done with the text and @shiryz has translated we will also need to create tabs to switch to Arabic and Hebrew...

**Comment by jsms90** [comment]
> Should the location-specific info pages be merged with the new apply pages (probably putting the logistic info below the application info)

I'd say don't combine them. Put yourself in the shoes of someone coming to this website for the first time. Why might you be there, what might you want to know / find out?

1. What is this Founders & Coders thing?
2. What is the course like / when does it start?
3. How do I join?

2 & 3 are quite different motivations. If you come to the site knowing nothing about FAC, you prob want to find out about the course _before_ thinking about applying.

**Comment by rebeccaradding** [comment]
The issue we have is that the answer to #1 is in a separate part of the site...whereas the location-specific info is really just logistics.

**Comment by jsms90** [comment]
Yeah for sure. But the LEARN MORE section, is replacing some of the contents of "Our Story", right?

**Comment by rebeccaradding** [comment]
Yeah it is. Just the press links though since they are location specific.

https://foundersandcoders.com/programme/

This page is also out of date. The meetup site is only for London and the course dates only apply to London. We should remove them and just link to those logistics pages in here.

**Comment by rebeccaradding** [comment]
For that matter, this page should have current course facilitators (and frankly probably shouldn't have Dona haha) https://foundersandcoders.com/community/

**Comment by jsms90** [comment]
I'm just saying, this task is much more than just updating the copy :disappointed:

**Comment by rebeccaradding** [comment]
@jsms90 you're so right. How can I help?

**Comment by jsms90** [comment]
Nothing, I'm afraid....form a task group that's happy to do some proper sprint planning & then some dev work for free? :sweat_smile: 

Haha. No. It's all good. I could literally just update the pages that we've got, but the navigation is just so horrendous, that I'm worried about it putting applicants off.

Also not super familiar with SCSS or Harp, so just being slow with that :disappointed:

**Comment by jsms90** [comment]
### What is the course like / when does it start?

Currently split between the ["Programme" tab](https://foundersandcoders.com/programme/) and the campus-specific "course information" pages for [London](https://foundersandcoders.com/programme/course-information/london) and [Naz](https://foundersandcoders.com/programme/course-information/nazareth)

Programme should start with
> Founders & Coders welcomes applicants over 18 from all academic and professional backgrounds. We find that successful students are self-motivated, prepared to work long hours, and committed to collaborative working. Coding also requires a deep curiosity about how things work and a certain stubbornness in the face of repeated failure.

Change the campus-specific pages to include:
+ Social media accounts if/when location-dependent
+ "Learn more" section
+ "London/Nazareth Programme" section - location & directions, hours, etc.
  + Apply button at the bottom.

---

You can't get to the course info pages from the "Programme" page or anywhere (?) other than the homepage :confused: which makes this info pretty hidden :cry: 

I can update this copy for now, and we can worry about the navigation issues later?

---

### How do I join?

If I click on a tab called "Apply", I expect to find out exactly how to do that as quickly and easily as possible. If I want to find out about the pre-reqs before or after, that's up to me.

#### Apply page:
+ Create github account
+ Create blank repo for your website
+ Use github account to join FCC
+ Use github account to join Codewars
+ Paragraph about the pre-reqs from the google doc & link to ["How to prepare" page](#how-to-prepare)
+ If you want to find out more about each of our campuses before you choose... (links to each campus-specific info page)
+ "Join Nazareth" / "Join London" buttons for each campus, that link to the google forms

#### How to prepare
+ As in google doc, minus the "create a new account" bits

#### Campus-specific application pages
+ Application dates & process (beginning of campus-specific google docs)
+ APPLY button linking to google form

**Comment by rebeccaradding** [comment]
@jsms90 can you show me in the Google doc? I think it's simpler if you have to read the pre-req page before the apply page, because it covers the same information in the way I reformatted it. I don't like repeating those steps on the Apply page, as I think the amount of text becomes overwhelming.

**Comment by jsms90** [comment]
@rebeccaradding I wouldn't repeat those steps on the same page. I'm suggesting they be on 2 different pages. Google doc will def be easier :+1:

**Comment by rebeccaradding** [comment]
I meant on a different page. I only think we need to list those steps once, is all. On each apply page we just link back to the pre-req page if people need a reminder.

**Comment by jsms90** [comment]
Just opened a PR with your version, since I'd spent some time doing it that way already. Unless you know how to run my branch's version of the website from your terminal, I'm not sure it's going to help you much?

But it might be useful for other devs like @shiryz @emilyb7. Always good to be able to run it and see what the copy would actually look like on the website.

Besides, I can always iterate on that PR if we go with anything different. Otherwise, ready to make small changes and merge.

---

As for separation, you may be right :thinking: 

But as little barrier to making an application as possible = :+1:, right?
Especially given how daunting the pre-reqs can look.  


I'm assuming that the aim is still to encourage people to put their application in before starting the pre-reqs?  
>  We do not expect you finish this work before applying...provide us with working usernames...so that we can monitor your progress

**Comment by rebeccaradding** [comment]
@jsms90 do you want to briefly discuss this by phone? It might be faster...

**Comment by rebeccaradding** [comment]
@jsms90 agree that it's worth getting something up to show the Nazareth team on the call tomorrow.

**Comment by jsms90** [comment]
@rebeccaradding 

Question from joe public:
> How can I change invite only to public in a room ?

Gitter:
> You can't I'm afraid. We made that decision very specifically. Imagine joining a room you thought was private, and then all of a sudden it became public. Not cool.
>
> So you'll have to create a new room, by clicking on the Add a Room button and follow the instructions from there, it's pretty simple.
>
> See [gitterHQ/gitter#676](https://github.com/gitterHQ/gitter/issues/676)

Want me to create 2 new rooms:
+ London
+ Nazareth
?

**Comment by rebeccaradding** [comment]
@jsms90 yes two new rooms. We should probably create a separate issue for London and Nazareth programmes announcing this change...as not everyone will have seen our work on the application/selection proposal.

For Gaza they have huge C4E attendance in person and can add people who attend manually, also their room needs to be private as they have to monitor who participates for political reasons (i.e. no visitors from Israel).

**Comment by shiryz** [comment]
we have 2 student agreements, every application has its own but the info contains london's.
should we move it to London's `apply` or delete it all together since it's already in the application form?

**Comment by shiryz** [comment]
@rebeccaradding, any news on visas that we can add to the naz info?

**Comment by jsms90** [comment]
@shiryz 
> we have 2 student agreements, every application has its own but the info contains london's.
should we move it to London's apply or delete it all together since it's already in the application form?

Is this in reaction to the copy which is sitting in the top comment? I pasted that in from the `selection-committee`, thinking it was for the website rather than the google doc application forms.

If you have a look at the current PR going into the website, the student agreement isn't mentioned anywhere.

**Comment by shiryz** [comment]
@jsms90, is the doc still relevant or should I move the comments to the pr?

**Comment by jsms90** [comment]
It's easier to move around large chunks of copy on the google doc. Plus, @rebeccaradding can't run this branch and have a look at what the copy looks like on the website itself.

Also time zones make it difficult...

I'm going to go ahead and put my latest suggestions into this PR. Then hopefully we can all comment on the PR from there. And you and I can be Rebecca's eyes :wink:

**Comment by rebeccaradding** [comment]
@jsms90 this is all done, no?
